LOAN/LoansTemporal/LoanApplicationsTemporal.rdf

```diff
@@ -79,34 +86,76 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-tem-app;LoanApplicant"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-arr-com;Applicant"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-app;appraisedBy"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isProvidedBy"/>
 				<owl:someValuesFrom rdf:resource="&fibo-loan-tem-app;LoanAppraiser"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-tem-app;mayBecome.1"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-loan;LoanDrawdown"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
 		<owl:disjointWith rdf:resource="&fibo-loan-tem-ph;DisbursedLoan"/>
 		<owl:disjointWith rdf:resource="&fibo-loan-tem-ph;RepaidLoan"/>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-tem-app;LoanApplicant">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-com;Applicant"/>
+	<owl:Class rdf:about="&fibo-loan-tem-app;BorrowerInformation">
```

Comment:
Should this not be ApplicantInformation?

Reply:
Maybe - but let's deal with it in another pass.  I'd like to get the most important things done, and then hopefully eliminate some of these additional ontologies unless there is significant value added.